ios/RNApplePay.m

```diff
@@ -119,6 +131,25 @@ - (NSArray *_Nonnull)getSupportedNetworks:(NSDictionary *_Nonnull)props
     return paymentSummaryItems;
 }
 
+- (PKContact *_Nonnull)getBillingContact:(NSDictionary *_Nonnull)props
+{
+    PKContact *contact = [[PKContact alloc] init];
+
+    NSDictionary *rnContact = props[@"billingContact"];
```

Comment:
I think it should be named "billingContact" and this prop will be optional. That's why you have to add check for it. Also it should be added to index.d.ts declaration file and it should have an example case.